lib/src/header_parser/sub_parsers/compounddecl_parser.dart

```diff
@@ -300,16 +300,25 @@ int _compoundMembersVisitor(clang_types.CXCursor cursor,
         // otherwise they will be added in the next iteration.
         if (!cursor.isAnonymousRecordDecl()) break;
 
+        // From the C11 spec:
```

Comment:
Should I spare the details here and instead say:

```
// Anonymous members are always unnamed. To avoid environment-dependent naming issues with the generated code, we explicitly use the empty string as spelling.
```

Reply:
Sure

Reply:
done